drivers/overlay/overlay.go

```diff
@@ -292,6 +292,15 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 		backingFs = fsName
 	}
 
+	runhome := filepath.Join(options.RunRoot, filepath.Base(home))
```

Comment:
`runhome` might not exist at this point.  You'd need to move the `idtools.MkdirAllAs` call before using it

Reply:
Fixed, thanks!